frontend/src/lib/components/HedgehogBuddy/HedgehogBuddy.stories.tsx

```diff
@@ -11,8 +11,7 @@ export default meta
 
 export const TheHedgehog: StoryFn<typeof MyHedgehogBuddy> = () => {
     return (
-        // eslint-disable-next-line react/forbid-dom-props
-        <div style={{ height: 200 }}>
+        <div className="h-[200px]">
```

Comment:
Let's keep things standard here too:
```suggestion
        <div className="h-52">
```